cmd/fyne_settings/settings/appearance.go

```diff
@@ -128,3 +207,74 @@ func (s *Settings) saveToFile(path string) error {
 
 	return ioutil.WriteFile(path, data, 0644)
 }
+
+type colorButton struct {
```

Comment:
The color picker I'm working on also defines a colorbutton, albeit slightly different, perhaps we should converge on a design as this would be a useful exported widget?